tracee-rules/engine/engine.go

```diff
@@ -102,12 +102,11 @@ func (engine *Engine) consumeSources(done <-chan bool) {
 					}
 					for _, sel := range se {
 						if sel.Source == "tracee" {
-							sig.OnSignal(types.SignalSourceComplete("tracee"))
+							sig.OnSignal(types.SignalSourceComplete("tracee")) // TODO: Handle error?
```

Comment:
this is intentional because if the sig errors we just ignore it. if you want to log the error that's fine as well

Reply:
Dropping it is fine by me as it's not logic that belongs to the engine anyway (it's a signature error). I'll make that change.